hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java

```diff
@@ -165,15 +165,15 @@ private void handleChangeClusterVersion(HttpPostCommand command) throws Unsuppor
                 res = res.replace("${STATUS}", "forbidden");
                 res = res.replace("${VERSION}", "null");
             } else {
-                Version version;
+                ClusterVersion version;
                 try {
-                    version = Version.of(versionParam);
+                    version = ClusterVersion.of(versionParam);
                     clusterService.changeClusterVersion(version);
                     res = res.replace("${STATUS}", "success");
-                    res = res.replace("${VERSION}", clusterService.getClusterVersion().asString());
+                    res = res.replace("${VERSION}", clusterService.getClusterVersion().toString());
                 } catch (Exception ex) {
                     res = res.replace("${STATUS}", "fail");
-                    res = res.replace("${VERSION}", clusterService.getClusterVersion().asString());
+                    res = res.replace("${VERSION}", clusterService.getClusterVersion().toString());
                 }
```

Comment:
This was on purpose asString, since toString may be more like a debug info or sth changed by someone in the future, not being aware it's being shown to the user in GUI. but feel free to decide by yourself.